src/modding/ModManager.cs

```diff
@@ -386,6 +559,11 @@ public override void _Ready()
 
         UpdateSelectedModInfo();
 
+        // The tab title has to be set here as they are normally set by node name in TabContainer
+        // Which means they won't be translated at all
+        modLoaderContainer.SetTabTitle(0, TranslationServer.Translate("MOD_LOADER_TAB"));
+        modLoaderContainer.SetTabTitle(1, TranslationServer.Translate("MOD_ERRORS_TAB"));
```

Comment:
These may also need to be called whenever language changes using `NotificationTranslationChanged.`